packages/remirror__extension-text-color/src/text-color-extension.ts

```diff
@@ -115,6 +115,18 @@ export class TextColorExtension extends MarkExtension<TextColorOptions> {
     return this.store.commands.removeMark.original({ type: this.type, ...options, expand: true });
   }
 
+  /**
+   * Get the color for the selection
+   */
+  @helper()
+  getTextColorForSelection(selection?: Selection): Helper<string> {
```

Comment:
We should default to the current selection here, I believe

```ts
import { Selection } from '@remirror/core';
```

```suggestion
  getTextColorForSelection(selection: Selection = this.store.getState().selection): Helper<string> {
```